packages/connectivity_plus/connectivity_plus_platform_interface/lib/src/utils.dart

```diff
@@ -1,22 +1,23 @@
 import 'package:connectivity_plus_platform_interface/connectivity_plus_platform_interface.dart';
 
-/// Convert a String to a ConnectivityResult value.
-ConnectivityResult parseConnectivityResult(String state) {
-  switch (state) {
-    case 'bluetooth':
-      return ConnectivityResult.bluetooth;
-    case 'wifi':
-      return ConnectivityResult.wifi;
-    case 'ethernet':
-      return ConnectivityResult.ethernet;
-    case 'mobile':
-      return ConnectivityResult.mobile;
-    case 'vpn':
-      return ConnectivityResult.vpn;
-    case 'other':
-      return ConnectivityResult.other;
-    case 'none':
-    default:
-      return ConnectivityResult.none;
-  }
+/// Convert a comma-separated String to a list of ConnectivityResult values.
```

Comment:
Small thing, but this comment seems incorrect, should be something like "Convert list of String connectivity states to list of ConnectivityResult values"